api/filesystem/filesystem_move_test.go

```diff
@@ -1,49 +1,35 @@
 package filesystem
 
 import (
-	"fmt"
 	"os"
 	"path"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-// temporary function until upgrade to 1.16
-func tempDir(t *testing.T) string {
-	tmpDir, err := os.MkdirTemp("", "dir")
-	assert.NoError(t, err, "MkdirTemp should not fail")
+func Test_movePath_shouldFailIfSourcePathDoesNotExist(t *testing.T) {
+	sourcePath := "missing"
+	desinationPath := t.TempDir()
 
-	return tmpDir
+	err := MoveDirectory(sourcePath, desinationPath)
+	assert.Error(t, err, "move directory should fail when shource path is missing")
```

Comment:
we should also ensure that on failure, the dir is where it should be - i'm presuming it should exist at the source location?

otherwise, we're only testing that the error return is right...

Reply:
`shource` spelling :)